docs/dc-apps-performance-toolkit-user-guide-bitbucket.md

```diff
@@ -47,9 +47,9 @@ Monthly charges will be based on your actual usage of AWS services, and may vary
 
 | Stack | Estimated hourly cost ($) |
 | ----- | ------------------------- |
-| One Node Bitbucket DC | 1 - 1.3 |
-| Two Nodes Bitbucket DC | 1.5 - 1.8 |
-| Four Nodes Bitbucket DC | 2.1 - 2.5 |
+| One Node Bitbucket DC | 1.03 |
```

Comment:
pls use a range of pricing. This way there is greater chance that real price would match our estimate.

Reply:
done